### PR TITLE
feat(release.yml): add repos from `hack/chart-testing/ct.yaml`

### PR DESCRIPTION
**Pull request by eyenx**
# Description

Instead of mantaining the repos ourselfes in the `release.yaml` we should use a script to read from `hack/chart-testing/ct.yaml`

# Issues

#197

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Comment by sinicis** [comment]
code itself LGTM but CI's faling

**Comment by eyenx** [comment]
> code itself LGTM but CI's failing

not anymore.

**Review by sinicis — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -28,9 +28,9 @@ jobs:
 
       - name: Add dependency chart repos
         run: |
-          helm repo add adfinis https://charts.adfinis.com && \
-          helm repo add secrets-store-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts && \
-          helm repo add bitnami https://charts.bitnami.com/bitnami
+          yq e '."chart-repos"[]' chart-testing/ct.yaml | awk 'BEGIN { FS = "=" } ; {print $1,$2}' | while read repo
```
Comment (vmaillot) [inline .github/workflows/release.yaml:31]: Is that `e` after `yq` a typo? Did you mean `-e`?

Comment (eyenx) [reply .github/workflows/release.yaml:31]: No its correct. Its short for eval.

Comment (eyenx) [reply .github/workflows/release.yaml]: We are using this yq: https://github.com/mikefarah/yq

**Review by vmaillot — APPROVED** [review]
LGTM

**Pull request merged**